docs/modules/ROOT/pages/how-tos/cloudscale/replace-storage-node.adoc

```diff
@@ -0,0 +1,77 @@
+= Replace a storage node
+
+:cloud_provider: cloudscale
+:kubectl_extra_args: --as=cluster-admin
+:delabel_app_nodes: yes
+:argo_app: rook-ceph
```

Comment:
Doesn't hurt, but I don't think we need to specify that (as we do not disable argocd)

Reply:
We actually use this page attribute in the text, e.g. in the `create-alertmanager-silence.adoc` partial.